iampolicymanagementv1/iam_policy_management_v1_integration_test.go

```diff
@@ -45,35 +45,44 @@ var _ = Describe("IAM Policy Management - Integration Tests", func() {
 		config       map[string]string
 		configLoaded bool = false
 
-		testAccountID     string
-		etagHeader        string = "ETag"
-		testPolicyETag    string = ""
-		testV2PolicyETag  string = ""
-		testPolicyId      string = ""
-		testV2PolicyId    string = ""
-		testUserId        string = "IBMid-GoSDK" + strconv.Itoa(rand.Intn(100000))
-		testViewerRoleCrn string = "crn:v1:bluemix:public:iam::::role:Viewer"
-		testEditorRoleCrn string = "crn:v1:bluemix:public:iam::::role:Editor"
-		testServiceName   string = "iam-groups"
-
-		testCustomRoleId                   string = ""
-		testCustomRoleETag                 string = ""
-		testCustomRoleName                 string = "TestGoRole" + strconv.Itoa(rand.Intn(100000))
-		testServiceRoleCrn                 string = "crn:v1:bluemix:public:iam-identity::::serviceRole:ServiceIdCreator"
-		testPolicyTemplateID               string = ""
-		testPolicyS2STemplateID            string = ""
-		testPolicyS2STemplateVersion       string = ""
-		testPolicyS2SUpdateTemplateVersion string = ""
-		testPolicyTemplateETag             string = ""
-		testPolicyTemplateVersion          string = ""
-		testPolicyAssignmentId             string = ""
-		examplePolicyTemplateName          string = "PolicySampleTemplateTestV1"
-		assignmentPolicyID                 string
-		testTargetAccountID                string = ""
-		assignmentRequesterId              string = "IBMid-" + strconv.Itoa(rand.Intn(100000))
-		assignmentID                       string = "orchestrator-id"
-		testPolicyAssignmentETag           string = ""
-		testTargetType                     string = "Account"
+		testAccountID      string
+		etagHeader         string = "ETag"
+		testPolicyETag     string = ""
+		testV2PolicyETag   string = ""
+		testPolicyId       string = ""
+		testV2PolicyId     string = ""
+		testUserId         string = "IBMid-GoSDK" + strconv.Itoa(rand.Intn(100000))
+		testViewerRoleCrn  string = "crn:v1:bluemix:public:iam::::role:Viewer"
+		testViewerRoleCrns string = "crn:v1:bluemix:public:iam::::role:Operator"
```

Comment:
Just this odd variable name. Feel free to correct me if I'm wrong, but wouldn't this suggestion make more sense?
```suggestion
		testOperatorRoleCrn string = "crn:v1:bluemix:public:iam::::role:Operator"
```